net/bridge.go

```diff
@@ -439,13 +439,8 @@ func configureIPTables(config *BridgeConfig) error {
 		// Steer traffic via the NPC
 		_ = ipt.NewChain("filter", "WEAVE-NPC") // ignore error because it might already exist
 		// If WEAVE-NPC chain doesn't exist then next line will fail
-		if err = ipt.AppendUnique("filter", "FORWARD", "-o", config.WeaveBridgeName, "-j", "WEAVE-NPC"); err != nil {
-			return err
-		}
-		if err = ipt.AppendUnique("filter", "FORWARD", "-o", config.WeaveBridgeName, "-m", "state", "--state", "NEW", "-j", "NFLOG", "--nflog-group", "86"); err != nil {
-			return err
-		}
-		if err = ipt.AppendUnique("filter", "FORWARD", "-o", config.WeaveBridgeName, "-j", "DROP"); err != nil {
+		// Use insert rather than append to try to get ahead of any ACCEPT rules which would defeat our policies
+		if err = insertUnique(ipt, "filter", "FORWARD", 1, "-o", config.WeaveBridgeName, "-j", "WEAVE-NPC"); err != nil {
```

Comment:
Maybe we could add a comment in the rule saying "NOTE: this must be before -j KUBE-FORWARD rule"